configs/mocha-config-devtools/react.test.ts

```diff
@@ -0,0 +1,8 @@
+import assert from 'assert';
+
+describe('mocha-config-devtools/react', function () {
+  it('has jsdom', function () {
+    assert(!!globalThis.window);
```

Comment:
A bit too basic, but just wanted to make sure globals are still exposed after a major version bump